src/http-gateways/path-gateway.md

```diff
@@ -242,6 +242,16 @@ These are the equivalents:
 When both `Accept` HTTP header  and `format` query parameter are present,
 `Accept` SHOULD take precedence.
 
+A Client SHOULD include the `format` query parameter in the request URL, in
+addition to the `Accept` header. This provides the best interoperability and
+ensures consistent HTTP cache behavior across various gateway implementations.
+
+A Gateway SHOULD include the
+[`Content-Location`](#content-location-response-header) header in the response when:
+- the request contains an `Accept` header specifying a well-known response
+  format, but the URL does not include the `format` query parameter
+- the `format` parameter is present, but does not match the format from `Accept`
```

Comment:
@hacdias I've added these clarifications as SHOULDs, the goal here is to push everyone towards including explicit `?format` in requests – haves a lot of headache in debugging interop/caching problems. 

We've always been doing this internally  (including both `format` and `Accept`), this writes it down as suggested practice.

Reply:
Is it problematic then that some information in accept headers cannot be expressed in the format parameters (e.g. dfs and duplicates for CARs)

Reply:
Potentially, could cause issues at some point in the future when a single gateway supports something more than "the Kubo default" and clients actually request non-default.

To avoid cache override problems, we add `car-*` URL params in #472 and use in `Content-Location` in https://github.com/ipfs/boxo/pull/603